DataProducts/inc/CrystalId.hh

```diff
@@ -0,0 +1,38 @@
+#ifndef DataProducts_CrystalId_hh
+#define DataProducts_CrystalId_hh
+//
+// Offline identifier of one calorimeter crystal channel
+//
+
+#include "Offline/DataProducts/inc/CaloConst.hh"
+#include <iosfwd>
+
+namespace mu2e {
+
+  class CrystalId{
+
+  public:
+
+    using value_type=std::uint16_t;
+
+    explicit CrystalId(value_type id) { _id = id; }
```

Comment:
If these Id types will ever be in a data product, root requires a default c'tor.